lib/osv/impact.py

```diff
@@ -123,8 +123,12 @@ def get_affected(repo, regress_commit_or_range, fix_commit_or_range):
   for commit in regress_commits:
     tags_with_bug.update(get_tags_with_commits(repo, [commit]))
 
-  tags_with_fix = get_tags_with_commits(repo, fix_commits)
+  if not regress_commits:
+    # If no introduced commit provided, assume all commits prior to fix are
+    # vulnerable.
+    tags_with_bug.update(get_all_tags(repo))
```

Comment:
aren't we adding all tags and not just tags before fix ?

Reply:
Yes, but then we compute `tags_with_bug - tags_with_fix` to get the actual list of tags.